.github/workflows/main.yml

```diff
@@ -171,10 +171,17 @@ jobs:
           path: ./*
           key: ${{ runner.os }}-${{ github.sha }}-build
 
-      - name: Compress artifacts
+      - name: Compress artifacts to zip file
         working-directory: dist
         run: zip -r leaflet.zip .
 
+      - name: Create tarball for NPM install
+        run: |
```

Comment:
To create a tarball I think it would be better to use [`npm pack`](https://docs.npmjs.com/cli/v8/commands/npm-pack) which creates a package exactly as NPM expects it.

Reply:
Oh nice! I didn't know about that, I'll amend...

Reply:
@jonkoops so long story short I don't think this is going to be so easy 😬 

Just doing `npm pack` on a fresh clone appears to work, but using the `.tgz` causes:
```
Attempted import error: 'leaflet' does not contain a default export (imported as 'L').
```

... which I _assume_ is because the actual build uses `rollup`.

If I `npm build` first, then `npm pack` I get https://github.com/Esri/esri-leaflet/issues/1200:

```
./node_modules/esri-leaflet/src/Layers/FeatureLayer/FeatureGrid.js
Attempted import error: 'toLatLngBounds' is not exported from 'leaflet' (imported as 'latLngBounds').
```

... any ideas? 😬 

---

I know my PR as given (using `tar` directly) isn't _great_, but it does work 😇